server/serverbase.js

```diff
@@ -104,10 +104,18 @@ exports.initialize2 = function(app) {
       }
     }
   }
+  function loadThumb() {
+    return function(req, res, next) {
+      req.url = path.dirname(req.url) + '/.thumbs/' +
+                path.basename(req.url);
+      staticUserData(req, res, next);
```

Comment:
Actually I couldn't find a way to upload a image, except for putting it directly in the database. Generating thumbnails for images can be done just on the server side, which is the way it is usually done, instead of generating it on the browser side and upload two pictures. If we want to support that we can definitely have a separate handle function to handle image upload, and decide what name to give the thumbnail too!